src/app/page.tsx

```diff
@@ -0,0 +1,60 @@
+import Image from "next/image";
+import { ChecklistService } from "../services/checklist";
+import ChecklistBoard from "@/images/checklist-board.png";
+import { CardList } from "../components/cardList";
+import { generateSlug } from "./utils";
+import { LocalesDropdown } from "@/components/localeDropdown";
+
+const HomePage = ({ locale = "en" }: { locale: string }) => {
+    const checklistService = ChecklistService.getInstance();
+
+    const checklists =
+        ChecklistService.getInstance().getChecklistsGroupedByCategory(locale);
+    const allAvailableLocales = checklistService.getAllChecklistLocales();
+
+    const Categories = Object.entries(checklists).map(
+        ([categoryName, checklists]) => (
+            <CardList
+                key={categoryName}
+                listName={categoryName}
+                listLink={`/category/${generateSlug(categoryName)}`}
+                checklists={checklists}
+                locale={locale}
+            />
+        )
+    );
+
+    return (
+        <div className="flex flex-col">
+            <div className="ml-auto">
+                <LocalesDropdown availableLocales={allAvailableLocales} />
+            </div>
+            <div className="flex justify-between items-center my-28">
+                <div className="max-w-xl">
+                    <h1 className="text-4xl font-mono">
+                        Always deliver your very best,{" "}
```

Comment:
Do we need these {" "}?

Reply:
Yeah, HTML trims trailing whitespace
Without it, this renders as `Always deliver your very best,always check`
The {" "} ensures it's `Always deliver your very best, always check`

Reply:
I thought the newline would be considered whitespace here. That's OK.

Reply:
There may be a way to make the newline be considered a whitespace by changing `display` or `box-sizing` via css
But just using `{" "}` makes it simpler
PS: This is auto added by prettier on formatting, that's why it's so common to see this on projects while no one seems to actually want to use it

Reply:
It's fine, thanks!